rootfs/etc/snmp/snmptrapd.conf

```diff
@@ -0,0 +1,3 @@
+disableAuthorization yes
+authCommunity log,execute,net COMMUNITYSTRING
```

Comment:
It will be good to have credentials as runtime inputs to avoid rebuilding the image, maybe environment variables?

Reply:
Yepp. This was just copy paste from https://docs.librenms.org/Extensions/SNMP-Trap-Handler/ , but I've added support for support for substituting COMMUNITYSTRING with $LIBRENMS_SNMP_COMMUNITY.